src/Controller/OcrController.php

```diff
@@ -20,11 +20,21 @@ class OcrController extends AbstractController
     /** @var Intuition */
     protected $intuition;
 
-    /** @var EngineBase */
+    /** @var TesseractEngine|GoogleCloudVisionEngine */
```

Comment:
Had to do this to appease Phan, otherwise it complains about calling the Engine-specific setters like `setPsm()`